modules/installation-special-config-butane.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-butane_{context}"]
+= Creating machine configs with Butane
+
+Machine configs are used to deploy control plane and worker machines by telling machines how to create users and filesystems, set up the network, install systemd units, and more.
```

Comment:
I think using "deploy" here is the wrong verb.  When I think of deploying machines, I think of either installing the cluster from scratch, scaling up MachineSets, or manually adding nodes to the cluster.

I would prefer to say that machine configs are used to *configure* machines.

Reply:
Good clarification...resolved.